scenarios/kubernetes_verify.py

```diff
@@ -27,12 +27,12 @@
 
 
 VERSION_TAG = {
-    '1.10': '1.10-v20180221-3655e24ae',
-    '1.11': '1.11-v20180504-30872f7f6',
-    '1.12': '1.12-v20180918-e0a9aa399',
-    '1.13': '1.13-v20181105-ceed87206',
+    '1.10': '1.10-v20181218-db74ab3f4',
```

Comment:
this might break everything.

Reply:
I *think* golint should work across all versions now, worth figuring out.